javascript-local-storage/data.js

```diff
@@ -0,0 +1,12 @@
+/* exported todos */
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+window.addEventListener('beforeunload', beforeUnload);
+function beforeUnload(event) {
+  var todosJSON = JSON.stringify(todos);
+  localStorage.setItem('javascript-local-storage', todosJSON);
+  if (previousTodosJSON !== null) {
+    todos = JSON.parse(previousTodosJSON);
+  }
```

Comment:
This conditional statement should be outside of the event listener. Otherwise the app won't actually reload the user's todos.